test-utils/Cargo.toml

```diff
@@ -24,6 +24,6 @@ edition = "2021"
 
 [dependencies]
 arrow = { version = "28.0.0", features = ["prettyprint"] }
-datafusion-common = { path = "../datafusion/common", version = "14.0.0" }
```

Comment:
No need to specify the version here since we do not release the `test-utils` crate

Reply:
I think technically there's no need for these redundant version numbers anywhere now that I fixed the docker script.